deployment/utils/gcb-discord-notifications/index.js

```diff
@@ -0,0 +1,95 @@
+// This is intended to be deployed as a Google Cloud Function to relay
+// Google Build events to Discord
+
+const octokit = require('@octokit/rest')()
+const Discord = require('discord.js')
+
+const hook = new Discord.WebhookClient(
+  process.env.WEBHOOK_ID,
+  process.env.WEBHOOK_TOKEN
+);
+
+// Subscribe is the main function called by Cloud Functions.
+module.exports.subscribe = async (event, callback) => {
+  const build = eventToBuild(event.data);
+
+  const status = ['WORKING', 'SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
+  if (status.indexOf(build.status) === -1) {
+    return;
+  }
+
+  // Send message to Discord.
+  const webhook = await createDiscordWebhook(build);
```

Comment:
paranoia:  test for webhook.message not being undefined before calling hook.send, since if for whatever reason status.build is not something expected, I think createDiscordWebhook would return webhook.message undefined.